lib/senders/index.js

```diff
@@ -24,12 +23,49 @@ module.exports = function (config, log) {
       sms: config.sms
     }
   )
-  .then(
-    function (senders) {
-      var mailer = senders.email
-      mailer.sendVerifyCode = function (account, code, opts) {
-        return P.resolve(mailer.verifyEmail(
-          {
+  .then(senders => {
+    const ungatedMailer = senders.email
+    const configBounces = config.bounces || {}
+    const MAX_HARD = configBounces.hard && configBounces.hard.max || 0
+    const MAX_COMPLAINT = configBounces.complaint && configBounces.complaint.max || 0
+    const BOUNCE_TYPE_HARD = 1
+    const BOUNCE_TYPE_COMPLAINT = 3
```

Comment:
On one hand, it might be easier to "remember", since they're strings. On the other, remembering wrong is a subtle bug. 😢